spyder/app/tests/test_mainwindow.py

```diff
@@ -431,6 +431,45 @@ def test_get_help_combo(main_window, qtbot):
     qtbot.waitUntil(lambda: check_text(webpage, "arange"), timeout=6000)
 
 
+@pytest.mark.slow
+@pytest.mark.skipif(PY2, reason="Invalid definition of function in Python 2.")
+def test_get_help_ipython_console_dot_notation(main_window, qtbot, tmpdir):
+    """
+    Test that Help works when called from the IPython console
+    with dot calls i.e np.sin
+
+    See spyder-ide/spyder#11821
+    """
+    shell = main_window.ipyconsole.get_current_shellwidget()
+    control = shell._control
+    qtbot.waitUntil(lambda: shell._prompt_html is not None,
+                    timeout=SHELL_TIMEOUT)
+
+    # Open test file
+    test_file = osp.join(LOCATION, 'script_unicode.py')
+    main_window.editor.load(test_file)
+    code_editor = main_window.editor.get_focus_widget()
+
+    # Run test file
+    qtbot.keyClick(code_editor, Qt.Key_F5)
+    qtbot.wait(500)
+
+    help_plugin = main_window.help
+    webview = help_plugin.rich_text.webview._webview
+    webpage = webview.page() if WEBENGINE else webview.page().mainFrame()
+
+    # Write function name
+    qtbot.keyClicks(control, u'np.sin')
+
+    # Get help
+    control.inspect_current_object()
+
+    # Check that a expected text is part of the page
+    qtbot.waitUntil(
+        lambda: check_text(webpage, "Trigonometric sine, element-wise."),
+        timeout=6000)
```

Comment:
Also, below this, please add a test for the aaʹbb function present in `script_unicode.py`, to test if we can show help for objects that contain unicode chars.

Reply:
See `test_mainwindow.test_get_help_ipython_console_special_characters`

Reply:
Ok, I haven't noticed that one :+1: